docs/DeveloperGuide.md

```diff
@@ -13,6 +13,27 @@ title: Developer Guide
 1. [AB-3 Developer Guide](https://se-education.org/addressbook-level3/DeveloperGuide.html)
 2. [PlantUML for sequence diagrams](https://plantuml.com/)
 
+---
+## Setting Up and Getting Started
+
+First, fork this repo, and clone the fork into your computer.
+
+If you plan to use Intellij IDEA (highly recommended):
+
+1. Configure the JDK: Follow the guide [se-edu/guides IDEA: Configuring the JDK](https://se-education.org/guides/tutorials/intellijJdk.html)  
+to ensure Intellij is configured to use JDK 11.
+2. Import the project as a Gradle project: Follow the guide
+[se-edu/guides IDEA: Importing a Gradle project](https://se-education.org/guides/tutorials/intellijImportGradleProject.html)
+to import the project into IDEA.
+:exclamation: Note: Importing a Gradle project is slightly different from importing a normal Java project.
+3. Verify the setup:
+   * Run AthletiCLI and try a few commands.
+   * Run the tests to ensure they all pass.
+
+[//]: # (What is the exact command to run the main?)
+
+
```

Comment:
Just to check is there an answer to this question here?

Reply:
No. I think I will be asking skylee...